test/functionalTest/cases/1126_GET_v2_subscriptions/GET_v2_all_subscriptions.test

```diff
@@ -178,7 +178,7 @@ Date: REGEX(.*)
                 "occupancy",
```

Comment:
PR https://github.com/telefonicaid/fiware-orion/pull/1388 (just merged) has solved the "9997 <-> LISTENER_PORT problem" so probably this will disapear from the PR after merge/rebase (maybe a conflict will arise).

NTC